### PR TITLE
Update README.md: way bigger warning

### PR DESCRIPTION
**Pull request by CrimsonGlory**
Bigger warning about the v2 breaking changes. At least temporarily. [skip ci]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ your application are configured.
 Once you have a Compose file, you can create and start your application with a
 single command: `docker compose up`.
 
+# About v2 update compatibility breaks
+
+Docker Compose v2 is a major version bump release of Docker compose. It has been completely rewriten from scratch in Golang (v1 was in Python). The installation instructions for Compose v2 differ from v1. v2 is not a standalone binary anymore and installation scripts will have to be adjusted. Some commands are different. For a smooth transition from legacy docker-compose 1.xx, please consider installing [compose-switch](https://github.com/docker/compose-switch) to translate `docker-compose ...` commands into Compose v2's `docker compose .... `. Also check v2's `--compatibility` flag.
```
Comment (ndeloof) [inline README.md:16]: rest of the documentation uses uppercase "V1" and "V2"
please place the compose-switch note as a separate paragraph to make it bit more visible to readers

File: README.md [hunk 2]
```diff
@@ -11,6 +11,10 @@ your application are configured.
 Once you have a Compose file, you can create and start your application with a
 single command: `docker compose up`.
 
+# About v2 update compatibility breaks
```
Comment (ndeloof) [inline README.md:14]: that's a bit of a frightening title!
I suggest "About update and backward compatibility"

**Review by ndeloof — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (CrimsonGlory) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (CrimsonGlory) [reply README.md]: lol, done

**Review by ndeloof — COMMENTED** [review]
for legal reasons we require contributors to Signed-off all commits, sorry for this
(git amend -s)

**Comment by CrimsonGlory** [comment]
> for legal reasons we require contributors to Signed-off all commits, sorry for this (git amend -s)

ups. I forgot. 
done

**Pull request merged**